### DISCUSSION TITLE
Tables not working in quarto presentations at browser used for developing

### DISCUSSION THREAD

**Discussion by kurianbenoy** [quarto-dev/quarto-cli#5069] [Q&A]
In Presentations build with RevealJS, I have been noticing the table are not rendering correctly, if I give below markdown:

```
## Sample table

| Right | Left | Default | Center |
|------:|:-----|---------|:------:|
|   12  |  12  |    12   |    12  |
|  123  |  123 |   123   |   123  |
|    1  |    1 |     1   |     1  |
```

![image](https://user-images.githubusercontent.com/24592806/229367369-c2f32dc4-4865-4ea1-ab56-9b27a3d062c2.png)

**Comment by mcanouil** [comment]
It works just fine with 1.2.475 and current dev version (45f6955378250e8288a79cd176c2dce92b279d2b).

Could you share more information (_e.g._, Quarto version, OS, _etc._), and a more complete working example rather than just the markdown table?

````qmd
---
title: "Quarto Minimal Working Example"
format: revealjs
---

## Sample table

| Right | Left | Default | Center |
|------:|:-----|---------|:------:|
|   12  |  12  |    12   |    12  |
|  123  |  123 |   123   |   123  |
|    1  |    1 |     1   |     1  |
````

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/229372275-5dd025c0-886b-4df0-ad2c-5bfddab47f45.png">

**Reply by kurianbenoy** [reply]
In my local, I am using:

quarto version: 1.2.335
OS: Windows

I am facing this issue in this [presentation slide](https://kurianbenoy.com/presentations/fossasia2023/index.html#/whisper-models). You can find the source code to it [here](https://github.com/kurianbenoy/kurianbenoy-website/blob/master/presentations/fossasia2023/index.qmd)

**Reply by mcanouil** [reply]
I do not understand, what's the issue?
I do not see obvious issue beside maybe the size.
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/8896044/229422968-93cf8c77-62b1-4e8f-93a8-dc47766b0ae7.png">

Note: the screenshot is directly from your rendered version.

**Reply by kurianbenoy** [reply]
Interesting. In my local main browser, it still rendering as shown in above error message. When I tried it in chrome, or in another computer it seems to work fine.

I might sometimes need to clear some cache or something?

**Reply by mcanouil** [reply]
I don't know what kind of browser you are using, but for me all chromium based browser gave the proper display, as well as Safari on Mac and Windows.
So the "issue" is definitely on your hand with your browser and/or your setup regarding your browser.

**Reply by kurianbenoy** [reply]
My local browser is firefox. It was the browser I was using to develop slides. Unfortunately the issue persists in firefox.

![image](https://user-images.githubusercontent.com/24592806/229667651-e76cc784-025e-49d7-a5ce-966de55479b2.png)


I tried using in other browser, and in even other computers with firefox where it was working perfectly fine.

**Reply by mcanouil** [reply]
Unfortunately browser issue such as yours is completely out of scope here.
You'll be better off to lookup your issue on Stack for example.

**Reply by kurianbenoy** [reply]
Noted. Thanks for helping out @mcanouil .
